dhis-2/dhis-services/dhis-service-analytics/src/test/java/org/hisp/dhis/analytics/event/data/EventsAnalyticsManagerTest.java

```diff
@@ -139,7 +139,7 @@ public void verifyGetEventSqlWithProgramWithNoRegistration()
 
         String expected = "select psi,ps,executiondate,storedby,ST_AsGeoJSON(psigeometry, 6) as geometry,longitude,latitude,ouname,oucode,ax.\"monthly\",ax.\"ou\"  from "
             + getTable( programA.getUid() )
-            + " as ax where ax.\"monthly\" in ('2000Q1') and (ax.\"uidlevel0\" in ('ouabcdefghA') ) limit 101";
+            + " as ax where ax.\"monthly\" in ('2000Q1') and ax.\"ou\" in ('ouabcdefghA') limit 101";
```

Comment:
At this point, the query params has not been through the query planner, so the org unit dimension name is not populated properly, hence just using `ou` in this unit test.